tool.js

```diff
@@ -10,15 +10,19 @@ module.exports = function (options) {
 
     AWS.config.region = options.region;
     var s3 = new AWS.S3({
-        accessKeyId: options.key,
-        secretAccessKey: options.secret
+	params: {
+		Bucket: options.params.bucket
+	}
+        accessKeyId: options.accesKeyId,
+        secretAccessKey: options.secretAccesKeyId
     });
 
     var updateWebsiteIndex = function (indexFile) {
 
         var deferred = Q.defer();
-
-        var s3 = new AWS.S3({params: {Bucket: options.bucket}});
+	
+	// this killed everything.. missing credentials.
+        //var s3 = new AWS.S3({params: {Bucket: options.bucket}});
```

Comment:
Ok, sounds good, this is indeed duplicate. But please remove these commented lines.